.travis.yml

```diff
@@ -85,10 +97,17 @@ matrix:
         - php -S localhost:8888 -c ./.scripts/travis/local_php_server.ini index.php &
         - sleep 3 # give Web server some time to bind to sockets, etc
 
+    # Coverage
+    - php: 7.0
+      env: JOB_NAME=coverage VARIA=true
+      script:
+        - phpdbg -qrr ./vendor/phpunit/phpunit/phpunit --configuration ./phpunit.xml --coverage-clover coverage.clover
```

Comment:
if an error occurs in `phpunit` the result isn't uploaded https://docs.travis-ci.com/user/customizing-the-build/#Customizing-the-Build-Step

I suggest moving the next 2 steps (download and clover upload) to an `after-script` step

Reply:
We don't want results for failed tests. I am not allowing failures here.